src/inmanta/deploy/scheduler.py

```diff
@@ -1233,9 +1233,9 @@ def _send_events(
                     else f"Deploying because an event was received from {resource_id}"
                 ),
                 priority=priority,
-                # report no ongoing deploys because ongoing deploys can not capture the event. We desire new deploys
-                # to be scheduled, even if any are ongoing for the same intent.
-                deploying=set(),
+                # report no ongoing deploys for the given resources because ongoing deploys can not capture the event.
+                # We desire new deploys to be scheduled, even if any are ongoing for the same intent.
+                deploying=self._deploying_latest,
```

Comment:
Shouldn't we set force_deploy here ?
```suggestion
                deploying=self._deploying_latest,
                force_deploy = True,
```